from_mwt_ds/DataScience/vw_executor/loggers.py

```diff
@@ -109,16 +109,25 @@ class FileLogger(ILogger):
     def __init__(self,
                  path: Optional[Union[str, Path]],
                  level: str = 'INFO',
+                 reset: bool = False,
```

Comment:
maybe adding docstring would be good, to mention that is to rewrite the file and not other behavior regarding the scoping/sub logger functionality.

Reply:
yeah, let me add it.